topics/w6-live-programming.md

```diff
@@ -1,11 +1,70 @@
 # W6 - Live Programming
```

Comment:
The blog post should introduce the topic of "live programming" before discussing the specific tool, SnipPy. The focus should be on explaining live programming to the reader, with SnipPy presented as an example rather than the main subject.

To improve the structure, consider organizing the post around these key points:

- What is live programming?
- What are the state-of-art applications in live programming?
- What challenges exist in live programming?
- How does SnipPy address these challenges?
- What are other challenges or limitations that still remain?
- What can be a potential direction to solve these challenges?